src/vstest.console/Resources/Resources.resx

```diff
@@ -336,7 +336,7 @@
     <value>The --ParentProcessId|/ParentProcessId argument requires the process id which is an integer. Specify the process id of the parent process that launched this process.</value>
   </data>
   <data name="InvalidPlatformType" xml:space="preserve">
-    <value>Invalid platform type:{0}. Valid platform types are x86, x64 and Arm.</value>
+    <value>Invalid platform type:{0}. Valid platform types are x86, x64, Arm and Arm64.</value>
```

Comment:
capitalize ARM and ARM64 in messages?

Reply:
Nope! This is the correct capitalization

Reply:
I would use lower case only as in C# compiler usage help. In any case the two lists below should be consistent, and they are not (@nohwnd): https://github.com/microsoft/vstest/blob/b99950648b6cd0a34fcf83137e09492021814ff2/src/Microsoft.TestPlatform.ObjectModel/Architecture.cs#L11-L16 https://github.com/microsoft/vstest/blob/b99950648b6cd0a34fcf83137e09492021814ff2/src/vstest.console/Resources/Resources.resx#L452